src/utils/getGnss.ts

```diff
@@ -50,7 +50,7 @@ const getTime = (location: Location_6): number => location['5'] * 1000
  * @see {@link ../../documents/gnss.md}
```

Comment:
**Suggestion:** The function `getGnss` doesn't handle the case when `location` is undefined. This could lead to a runtime error if this parameter is not provided.
```suggestion
 export const getGnss = (location?: Location_6): GetGnssResult => {
     if (!location) {
         return { error: new Error('Missing required parameter: location') }
     }
     // ...
 }
```

Reply:
It is not true. The case is conver.